portal-web/test/functional/com/liferay/portalweb/macros/DMFolder.macro

```diff
@@ -251,6 +251,18 @@ definition {
 		AssertElementNotPresent(locator1 = "DocumentsAndMedia#DESCRIPTIVE_LIST_FOLDER_ICON");
 	}
 
+	macro downloadViaEllipsisMenu {
+		ProductMenu.gotoPortlet(
+			category = "Content & Data",
+			portlet = "Documents and Media");
+
+		Click(
+			key_dmFolderName = "${dmFolderName}",
+			locator1 = "DocumentsAndMedia#FOLDER_ELLIPSIS");
```

Comment:
Can you reuse Icon#ROW_VERTICAL_ELLIPSIS here? you would just need to declare the folder name